Core/Sources/Implementations/species.cpp

```diff
@@ -71,25 +71,26 @@ auto Hippocrates::Species::IsStagnant() const -> bool {
 
 auto Species::LetPopulationLive() -> void {
 	for (auto& organism : population) {
-		UpdateOrganism(organism);
+		EvaluateOrganism(organism);
```

Comment:
Misleading name. Definition of evaluate:
- to determine or set the value or amount of; appraise:
- to judge or determine the significance, worth, or quality of; assess:
- to determine or calculate the numerical value of (a formula, function, relation, etc.).

This implies that this has something to do with the fitness function, which it doesn't